CONTRIBUTING.md

```diff
@@ -37,7 +35,16 @@ This is a rough outline of what a contributor's workflow looks like:
 - Make sure your commit messages are in the proper format (see below).
 - Push your changes to a topic branch in your fork of the repository.
 - Make sure the tests pass, and add any new tests as appropriate.
-- Submit a pull request to the original repository.
+- Submit a pull request to the original repository and assign it to yourself.
```

Comment:
Rechecked, according to https://help.github.com/articles/repository-permission-levels-for-an-organization/ one needs write, admin or owner permissions, hence removing that part.